test/shared_setup.py

```diff
@@ -127,29 +127,30 @@ def test_action_target(self):
         """Test action_target function, mostly w.r.t. backward compatibility."""
         def fake_setup(*args, **kwargs):
             """Fake setup function to test action_target with."""
-            print 'args: ', args
-            print 'kwargs:', kwargs
+            print('args: ', args)
+            print('kwargs:', kwargs)
```

Comment:
This means trouble in Python 2 I think, and it may be why the test for this is failing in Python 3!

Should be this, to avoid that you're printing a tuple:
```python
print('args: %s' % args)
print('kwargs: %s' % kwargs)
```

Reply:
if you put this at the top of the file `from __future__ import print_function` then print() will behave the same in python2 as in python3